tutorials/Window_API_new/README.md

```diff
@@ -0,0 +1,492 @@
+# OS windows management (new Composable API, experimental)
+
+## What is covered
+
+In this tutorial we will show you how to work with windows using Compose for Desktop.
```

Comment:
Maybe couple more sentences like those:

We represent window state in the shape suitable for Compose-style state manipulations and automatically mapping it into the operating system window state. So top level windows can be both conditionally created in other composable functions and their window manager state could be manipulated using state produced by `rememberWindowState()` function.

Reply:
Thanks! This is much better :).